src/templates/artworks.js

```diff
@@ -52,7 +50,7 @@ render() {
   const { frontmatter } = data.markdownRemark;
     return (
       <Layout className="container" data={data} location={this.props.location}>
-        <div style={{ marginBottom: rhythm(2) }}>
```

Comment:
I deleted this, because i want to remove completely typography.js from the template, i think has no sense with bulma. And it gave me swrong result with the Gallery component: the style `margin-bottom` for the `img` tag makes the fullscreen and play button stay very near to the image border.